server/src/main/java/org/apache/iotdb/db/engine/compaction/TsFileIdentifier.java

```diff
@@ -44,8 +44,8 @@
   public static final int LOGICAL_SG_OFFSET_IN_LOG = 0;
   public static final int VIRTUAL_SG_OFFSET_IN_LOG = 1;
   public static final int TIME_PARTITION_OFFSET_IN_LOG = 2;
-  public static final int SEQUENCE_OFFSET_IN_LOG = 3;
-  public static final int FILE_NAME_OFFSET_IN_LOG = 4;
+  public static final int SEQUENCE_OFFSET_IN_LOG = 4;
+  public static final int FILE_NAME_OFFSET_IN_LOG = 3;
```

Comment:
add a javadoc to warn others not modifying this.

Reply:
resolve